src/product.css

```diff
@@ -0,0 +1,84 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
```

Comment:
let's not set width to BEM-block

Reply:
You need to set width. Simply give this element another class (e.g. "main__card") and give margins and width to this class.